### PR TITLE
new pr template

### PR DESCRIPTION
**Pull request by Raycoms**
Closes #
Closes #
Closes #

# Changes proposed in this pull request:
- New PR template
-
-

Review please

### CODE REVIEW

**Comment by MotionlessTrain** [comment]
Just in case: does it matter I don't test all buildings before submitting a PR to change them?
I'm afraid I don't get to check that box often

**Comment by Raycoms** [comment]
> Just in case: does it matter I don't test all buildings before submitting a PR to change them? I'm afraid I don't get to check that box often

No, its fine, its just so we got an idea. Building PRs dont have to be tested

File: .github/PULL_REQUEST_TEMPLATE [hunk 1]
```diff
@@ -7,4 +7,7 @@ Closes #
 -
 -
 
+
+[ ] Yes I tested this before submitting it.
```
Comment (Nightenom) [inline .github/PULL_REQUEST_TEMPLATE:11]: well, better add sth like this:
a) tested in single new
b) tested in single existing
c) tested in multi

Comment (Raycoms) [reply .github/PULL_REQUEST_TEMPLATE:11]: I adjusted it.

**Comment by MotionlessTrain** [comment]
Aren't github templates just used from the default branch? It is now merged to version/1.29, which I don't think has an effect on pull requests at all

**Comment by Raycoms** [comment]
Ill merge it to the main too

**Pull request merged**